views/DCC_Tabbed_Sheet.css.erb

```diff
@@ -4377,4 +4377,26 @@ div[data-groupname^="repeating_mutations"] .repitem,
     max-width: 100%;
 }
 
+/* Handle Dark Mode */
+.sheet-rolltemplate-darkmode .sheet-RollTemp-wrapper {
+  background-color: white;
+}
+
+.sheet-rolltemplate-darkmode .inlinerollresult {
+  background-color: #fef68e !important;
+  border: 2px solid #fef68e !important;
+}
+
+.sheet-rolltemplate-darkmode .inlinerollresult.fullcrit {
+  border: 2px solid #3fb315 !important;
+}
+
+.sheet-rolltemplate-darkmode .inlinerollresult.fullfail {
+  border: 2px solid #b31515 !important;
+}
+
+.sheet-rolltemplate-darkmode .inlinerollresult.importantroll {
+  border: 2px solid #4a57ed !important;
+}
```

Comment:
All of these colors are from the base Roll20 CSS. They're all `!important`, as dark mode imports an additional stylesheet that overrides both the base CSS and the sheet CSS. I acknowledge that this is hacky, but it gets the job done.